pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -63,6 +63,13 @@ const (
 
 	deleteAfterAnnotation    = "hive.openshift.io/delete-after" // contains a duration after which the cluster should be cleaned up.
 	tryInstallOnceAnnotation = "hive.openshift.io/try-install-once"
+
+	platformAWS       = "aws"
+	platformAzure     = "azure"
+	platformGCP       = "hcp"
```

Comment:
Is "hcp" correct? Whether this value is "gcp"?

Reply:
yes, resolved as per last push.